.gitignore

```
@@ -58,3 +58,5 @@ webapp/analyzer.report.html
 user-password-change.txt
 user-password-change.csv
 /tests/helm/values.yaml
+*.png
```

we have other images in the repo, it can accidentally remove them. It's better to specify the folder 
```suggestion
/documentation/*.png
```